src/PiWeb.Import.Sdk/Modules/ImportFormat/MappingTarget.cs

```diff
@@ -0,0 +1,62 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2024                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+namespace Zeiss.PiWeb.Import.Sdk.Modules.ImportFormat;
+
+/// <summary>
+/// The well known entity roles that are available as mapping targets.

```

Comment:
Let's just say "The possible mapping targets of a mapping rule." Entity roles are currently not part of the domain structure of import plans in plugins.

Reply:
Done